src/generators/index.mjs

```diff
@@ -7,6 +7,8 @@ import manPage from './man-page/index.mjs';
 import legacyJson from './legacy-json/index.mjs';
 import legacyJsonAll from './legacy-json-all/index.mjs';
 import addonVerify from './addon-verify/index.mjs';
+import apiLinks from './api-links/index.mjs';
+import astJs from './ast-js/index.mjs';
 
 export default {
```

Comment:
IMO, the 'ast-js' generator should not be allowed to be used in the CLI... Or do we want to allow users to explicitly just call ast-js? I'm neutral about that, and that's fine IMO.